app/precompiled.py

```diff
@@ -162,15 +162,26 @@ def overlay_template():
     file_data = BytesIO(encoded_string)
 
     validate = request.args.get('validate') in ['true', '1']
-
-    return send_file(
-        filename_or_fp=overlay_template_areas(
-            file_data,
-            int(request.args.get('page', 1)),
-            not validate
-        ),
-        mimetype='image/png',
-    )
+    invert = request.args.get('invert') in ['true', '1']
```

Comment:
this isn't actually needed as it turns out this function isn't called from any of our other services

Reply:
also the validate request arg isn't used